.eslintrc.yml

```diff
@@ -24,9 +24,18 @@ rules:
   prefer-arrow-callback: error
   prefer-const: error
   react/no-deprecated: off
+  react/prop-types: off
   strict: error
   symbol-description: error
   yoda:
   - error
   - never
   - exceptRange: true
+overrides:
```

Comment:
You could put this in `website/playground/.eslintrc.yml` instead to avoid having to use the overrides.